docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -18,20 +20,41 @@ For Windows, or alternative installs, you can [download a Nodejs installer](http
 
 Here are the quick instructions to install using nvm, see the [full installation instructions](https://github.com/nvm-sh/nvm#installing-and-updating) for additional details.
 
```

Comment:
Here are the quick instructions for Windows and Mac to install using nvm, see the [full installation instructions](https://github.com/nvm-sh/nvm#installing-and-updating) for additional details.
(As we need to let the user know which OS we are giving quick instructions to.)

Reply:
The instructions also work for Linux, so it is for all.